### DISCUSSION TITLE
Set default note/folder Icon

### DISCUSSION THREAD

**Discussion by ZekiSNL** [FlorianWoelki/obsidian-icon-folder#505] [Ideas]
Just an additional option that allows the user to set an icon as default to use whenever a new folder or note is created. If you want to go crazy with it, you can make it so you can assign folders/note icons to folders cascading down.

Ex. Setting a folder named Towns to default all notes icons to a house. 

(Apologies if this is already a thing or brought up already. If it is, I wasn't able to find it.)

**Comment by FlorianWoelki** [comment]
Hey there!

It's already implemented in a more advanced way. You can find out more [here](https://florianwoelki.github.io/obsidian-iconize/files-and-folders/custom-rules.html) for setting a default icon or using custom rules.

**Reply by ZekiSNL** [reply]
Jeez I really am blind. Thank you!

**Reply by FlorianWoelki** [reply]
No worries at all :)
